### PR TITLE
Properly ignore first col of tables in scrape_term

### PR DESCRIPTION
**Pull request by madhavarshney**

### CODE REVIEW

File: scrape_term.py [hunk 1]
```diff
@@ -69,15 +69,19 @@ def parse(content, db):
         rows = t.find_all('tr', {'class': 'CourseRow'})
         s = defaultdict(lambda: defaultdict(list))
         for tr in rows:
-            cols = tr.find_all(lambda tag: tag.name == 'td' and not tag.get_text().isspace())
```
Comment (phi-line) [inline scrape_term.py:72]: 🏳️  🏳️  🏳️

File: scrape_term.py [hunk 2]
```diff
@@ -69,15 +69,19 @@ def parse(content, db):
...
+                # it is either empty or contains a "flag" icon
+                cols.pop(0)
+
                 for i, c in enumerate(cols):
                     a = c.find('a')
                     cols[i] = (a.get_text() if a else cols[i].get_text()).strip()
 
                 try:
-                    key = get_key(f'{cols[0] if cols[0] else cols[1]}')[0]
+                    key = get_key(cols[0])[0]
```
Comment (madhavarshney) [inline scrape_term.py:84]: Oh wait, not sure if this is necessary / what it breaks.

File: scrape_term.py [hunk 3]
```diff
@@ -69,15 +69,19 @@ def parse(content, db):
         rows = t.find_all('tr', {'class': 'CourseRow'})
         s = defaultdict(lambda: defaultdict(list))
         for tr in rows:
-            cols = tr.find_all(lambda tag: tag.name == 'td' and not tag.get_text().isspace())
+            cols = tr.find_all(lambda tag: tag.name == 'td')
 
             if cols:
+                # The first <td> is a field that is not relevant to us
+                # it is either empty or contains a "flag" icon
+                cols.pop(0)
```
Comment (phi-line) [inline scrape_term.py:77]: Wouldn't we want to do `cols = cols[1:]`?

`cols.pop(0)` would pop the last elem not the first

Comment (madhavarshney) [reply scrape_term.py:77]: We are popping the item at index = 0

Comment (phi-line) [reply scrape_term.py:77]: Oh derp 🤦

File: scrape_term.py (hunk 2 above)
Comment (phi-line) [reply scrape_term.py:84]: I think this is okay now because you popped the first.
Can you test locally to see if it returns a desired result?

Comment (madhavarshney) [reply scrape_term.py:84]: According to my testing, it seems to work fine. (Tested with LiveMyPortalData)

**Pull request merged**